.github/workflows/generate-artifacts.yml

```diff
@@ -0,0 +1,69 @@
+name: allcontributors-zenodo
+
+on:
+  pull_request: []
+
+jobs:
+  Update:
+    name: Generate
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+      - name: Setup test environment
+        run: conda create --quiet --name test requests
+
+      - name: Run Testing
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          export PATH="/usr/share/miniconda/bin:$PATH"
+          source activate test
+          /bin/bash tests/test_client.sh
+
+      - name: Generate Updated Zenodo and Contributors
+      
+        # Important! Update to release https://github.com/con/allcontributors-zenodo
+        uses: ./
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:        
+
+          # ".zenodo.json to update. If does not exist, must define zenodo_doi"
+          zenodo_file: .zenodo.json
```

Comment:
It technically is hard coded because it's the default for the variable (if the user doesn't define it). I show defining it so the user knows that they could, technically, if that is wanted. It would actually be reasonable to want to put the file somewhere else in the repo, e.g., `.github/.zenodo.json`.

Reply:
then it would not be picked up by zenodo upon release AFAIK (or am I wrong?), or why would you want to place it aside?

Reply:
well, may be only for testing... ok -- I am ok to consider it resolved ;)